go/vt/schemamanager/tablet_executor.go

```diff
@@ -441,6 +441,33 @@ func (exec *TabletExecutor) executeOnAllTablets(ctx context.Context, execResult
 	}
 }
 
+// applyAllowZeroInDate takes a SQL string which may contain one or more statements,
+// and, assuming those are DDLs, adds a /*vt+ allowZeroInDate=true */ directive to all of them,
+// returning the result again as one long SQL.
+func applyAllowZeroInDate(sql string) (string, error) {
+	// sql may be a batch of multiple statements
+	sqls, err := sqlparser.SplitStatementToPieces(sql)
+	if err != nil {
+		return sql, err
+	}
+	modifiedSqls := []string{}
```

Comment:
```suggestion
	var modifiedSqls []string
```

Static analysis will flag this (at least my IDE did!)